### PR TITLE
Indicate scope in application-specific prefix label

### PR DESCRIPTION
**Pull request by enygren**
Recommend indicating the scope (host, wildcard, or domain) as part of the application-specific prefix label.  Also include discussion of scopes within Security Considerations.

### CODE REVIEW

**Review by paulwouters — APPROVED** [review]

**Comment by ShivanKaul** [comment]
This makes sense to me. @shuque we've discussed this before, any thoughts?

**Comment by shuque** [comment]
Yes, I agree that it's an excellent idea to distinguish host vs wildcard scope in the DCV record. My main additional thought around this is: do we need to shop around this recommendation to ACME and broader? It would be good to ensure that we have general buy-in for this recommendation.

File: draft-ietf-dnsop-domain-verification-techniques.md [hunk 1]
```diff
@@ -214,6 +214,14 @@ Future specifications may provide better mechanisms or recommendations for defin
 
 The RECOMMENDED format is application-specific underscore prefix labels. Domain Control Validation records are constructed by the provider by prepending the label "`_<PROVIDER_RELEVANT_NAME>-challenge`" to the domain name being validated (e.g. "\_foo-challenge.example.com"). The prefixed "_" is used to avoid collisions with existing hostnames.
 
+### Scope Indication {#scope-indication}
+
+For applications that may apply more broadly than to a single host name, the RECOMMENDED approach is to differentiate the application-specific underscore prefix labels to also include the scope (see #scope). In particular:
+
+* "`_<PROVIDER_RELEVANT_NAME>-host-challenge.example.com`" applies only to the specific host name of "example.com" anything underneath it.
```
Comment (enygren) [inline draft-ietf-dnsop-domain-verification-techniques.md:221]:
```suggestion
* "`_<PROVIDER_RELEVANT_NAME>-host-challenge.example.com`" applies only to the specific host name of "example.com" and not to anything underneath it.
```

Fixing typo

**Comment by enygren** [comment]
I sent mail to acme@ietf.org for discussion on this.  Agreed that broader feedback would be worthwhile.  Would we want to bring this up on the dnsop list as well for confirmation, or just wait for when we publish the next version of the draft?

**Comment by paulwouters** [comment]
On Oct 3, 2023, at 09:32, Erik Nygren ***@***.***> wrote:﻿
I sent mail to ***@***.*** for discussion on this.  Agreed that broader feedback would be worthwhile.  Would we want to bring this up on the dnsop list as well for confirmation, or just wait for when we publish the next version of the draft?we are overdue for publishing 😀Paul

**Comment by shuque** [comment]
I think we can merge this in for now. Looks like we need to continue the discussion on this topic with ACME and see if we can get them to converge on this, but I don't see any harm in pushing this one out. We can always tweak later if needed.

**Review by shuque — APPROVED** [review]

**Pull request merged**